.husky/pre-commit

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+# Before adding version-manager code here, read https://typicode.github.io/husky/how-to.html#solution.
```

Comment:
Unrelated, but while setting up my new laptop I learned why people occasionally like to add a line here that activates nvm.